scss/mixins/display.scss

```
@@ -0,0 +1,11 @@
+@mixin displayFlexCenter($align: center, $justi: center, $gap: 0) {
+    display: flex;
+    justify-content: $justi;
+    align-items: $align;
+    gap: $gap;
+}
+
+@mixin displayFlexCol {
+    display: flex;
+    flex-direction: column;
```

Nên viết mixin bao quát tất cả thuộc tính, để page khác còn dùng lai
```suggestion
@mixin displayFlexCol($align: center, $justi: center, $gap: 0) {
    display: flex;
    flex-direction: column;
    justify-content: $justi;
    align-items: $align;
    gap: $gap;
}
```